#information of an object

point3d pos
float64 size
float64 r
int32 match
